pipelines/ops/ml-mongo-dump/Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+node() {
+    try {
+        String ANSI_GREEN = "\u001B[32m"
```

Comment:
None of these are required since you are not using these variables anywhere in the below code. Remove all `ANSI_` variable declaration.